pallets/order-book/src/lib.rs

```diff
@@ -345,6 +358,16 @@ pub mod pallet {
                 OrderBook::<T>::default_nft(order_book_id, dex_id)
             };
 
+            #[cfg(feature = "wip")] // order-book
```

Comment:
This pallet is already hidden by `wip` feature, so you can pass `wip` feature to `common` crate in `Cargo.toml`

Reply:
The build is failed for `cargo build`, because `LiquiditySourceType::OrderBook` is hidden by `wip`